### PR TITLE
Fixes #1790 Collapse state alt / title fix

### LINKED ISSUE #1790: Expcolimage in Board Statistic
Basically, this is not important but slightly disturbing. In this image --&gt; http://prntscr.com/5xllhq if I click [-], Board statistic will be slide up. It's work fine.Expcolimage will show [+]. 
And when I refresh the page with board statistic still slide up, it's work fine too. 
The problem is when my cursor pointing to expcolimage after refresh (still slide up), it will show [-] should be to [+]. 
It's happened in this forum too.

Original thread: [Expcolimage in Board Statistic](http://community.mybb.com/thread-166219.html)

### PR DESCRIPTION
**Pull request by effone**
https://github.com/mybb/mybb/issues/1790

### CODE REVIEW

File: install/resources/mybb_theme.xml [hunk 1]
```diff
@@ -4373,7 +4373,7 @@ $(".forumjump").change(function() {
 <thead>
```
Comment (Shade-) [inline install/resources/mybb_theme.xml]: Template versioning should be updated when editing templates. The next scheduled version is 1.8.16, so it should be updated to `1816`. I can't add a comment to the exact line which is 4372 (`index_boardstats`).

File: install/resources/mybb_theme.xml [hunk 2]
```diff
@@ -5989,7 +5989,7 @@ if(use_xmlhttprequest == "1")
 		<template name="misc_help_section" version="1800"><![CDATA[<table border="0" cellspacing="{$theme['borderwidth']}" cellpadding="{$theme['tablespace']}" class="tborder clear">
```
Comment (Shade-) [inline install/resources/mybb_theme.xml]: Same here.

File: install/resources/mybb_theme.xml [hunk 3]
```diff
@@ -7722,7 +7722,7 @@ if(use_xmlhttprequest == "1")
 		<template name="modcp_nav_banning" version="1800"><![CDATA[<tr><td class="trow1 smalltext"><a href="modcp.php?action=banning" class="modcp_nav_item modcp_nav_banning">{$lang->mcp_nav_banning}</a></td></tr>]]></template>
 		<template name="modcp_nav_forums_posts" version="1800"><![CDATA[	<tr>
```
Comment (Shade-) [inline install/resources/mybb_theme.xml]: Same here.

File: install/resources/mybb_theme.xml [hunk 4]
```diff
@@ -7739,7 +7739,7 @@ if(use_xmlhttprequest == "1")
 		<template name="modcp_nav_reportcenter" version="1800"><![CDATA[<tr><td class="trow1 smalltext"><a href="modcp.php?action=reports" class="modcp_nav_item modcp_nav_reports">{$lang->mcp_nav_report_center}</a></td></tr>]]></template>
 		<template name="modcp_nav_users" version="1800"><![CDATA[	<tr>
```
Comment (Shade-) [inline install/resources/mybb_theme.xml]: Here.

File: install/resources/mybb_theme.xml [hunk 5]
```diff
@@ -10323,7 +10323,7 @@ if(use_xmlhttprequest == "1")
 		<thead>
```
Comment (Shade-) [inline install/resources/mybb_theme.xml]: Here (`private_quickreply`).

File: install/resources/mybb_theme.xml [hunk 6]
```diff
@@ -11695,7 +11695,7 @@ if(use_xmlhttprequest == "1")
...
```
Comment (Shade-) [inline install/resources/mybb_theme.xml]: Here (`showthread_quickreply`).

File: install/resources/mybb_theme.xml [hunk 7]
```diff
@@ -11798,7 +11798,7 @@ if(use_xmlhttprequest == "1")
 <thead>
```
Comment (Shade-) [inline install/resources/mybb_theme.xml]: Here (`showthread_threadnotes`).

File: install/resources/mybb_theme.xml [hunk 8]
```diff
@@ -12758,7 +12758,7 @@ if(use_xmlhttprequest == "1")
 		<template name="usercp_nav_misc" version="1800"><![CDATA[<tbody>
```
Comment (Shade-) [inline install/resources/mybb_theme.xml]: Here.

File: install/resources/mybb_theme.xml [hunk 9]
```diff
@@ -12737,7 +12737,7 @@ if(use_xmlhttprequest == "1")
 		<template name="usercp_nav_messenger" version="1800"><![CDATA[<tbody>
```
Comment (Shade-) [inline install/resources/mybb_theme.xml]: Here.

File: install/resources/mybb_theme.xml [hunk 10]
```diff
@@ -12775,7 +12775,7 @@ if(use_xmlhttprequest == "1")
 		<template name="usercp_nav_profile" version="1800"><![CDATA[<tbody>
```
Comment (Shade-) [inline install/resources/mybb_theme.xml]: Here.

File: install/resources/mybb_theme.xml (hunk 1 above)
Comment (effone) [reply install/resources/mybb_theme.xml]: @Shade- I don't have push access to the PR as I have deleted the main fork along with this branch.
If you see its okay to merge I will create another back to back PR with correct revision numbers?
Is that okay? Else I have to create the complete PR fresh ...

Comment (Shade-) [reply install/resources/mybb_theme.xml]: As long as you reference this PR with the new one, it's ok.

**Pull request merged**